Docker does not work with ARM based Mac or Windows machines

Trying to follow the directions from the readme file:

```sh
docker build . -t dc-couchbase-agent:latest
```

This get's get's to step 7 and then fails:

#12 5.561 npm ERR! code 1
#12 5.561 npm ERR! path /app/node_modules/couchbase
#12 5.562 npm ERR! command failed
#12 5.562 npm ERR! command sh -c -- prebuild-install -r napi || cmake-js compile
#12 5.562 npm ERR! [ '/usr/local/bin/node', '/app/node_modules/.bin/cmake-js', 'compile' ]
#12 5.562 npm ERR! prebuild-install warn install No prebuilt binaries found (target=6 runtime=napi arch=arm64 libc=musl platform=linux)
#12 5.562 npm ERR! info TOOL Using Unix Makefiles generator.
#12 5.562 npm ERR! info DIST Downloading distribution files to: /root/.cmake-js/node-arm64/v16.19.0
#12 5.562 npm ERR! http DIST 	- https://nodejs.org/dist/v16.19.0/SHASUMS256.txt
#12 5.562 npm ERR! http DIST 	- https://nodejs.org/dist/v16.19.0/node-v16.19.0-headers.tar.gz
#12 5.562 npm ERR! info REP Build has been failed, trying to do a full rebuild.
#12 5.562 npm ERR! ERR! OMG CMake executable is not found. Please use your system's package manager to install it, or you can get installers from there: http://cmake.org.
#12 5.564
#12 5.564 npm ERR! A complete log of this run can be found in:
#12 5.564 npm ERR!     /root/.npm/_logs/2022-12-20T20_15_58_222Z-debug-0.log 

The Couchbase Node SDK for ARM64 only supports a few Linux platforms:
https://docs.couchbase.com/nodejs-sdk/current/project-docs/compatibility.html#arm-processor-support

I believe the issue could be with this.  From looking at the Dockerfile, it looks like the alpine image is used:

```sh
FROM node:16-alpine
```

Can we switch this over to Ubuntu 20.04?

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,4 @@
-FROM node:16-alpine
-RUN apk add g++ make py3-pip\
-    && rm -rf /var/cache/apk/*
+FROM node:16
 WORKDIR /app
 COPY package.json .
 COPY package-lock.json .

